apps/server/src/shared/infra/identity-management/keycloak/service/keycloak-identity-management.service.integration.spec.ts

```diff
@@ -1,109 +1,209 @@
-import { Configuration } from '@hpi-schul-cloud/commons/lib';
 import KeycloakAdminClient from '@keycloak/keycloak-admin-client';
 import { Test, TestingModule } from '@nestjs/testing';
-import { IdentityManagementService } from '../../identity-management.service';
-import { KeycloakSettings } from '../interface/keycloak-settings.interface';
+import { KeycloakAdministrationService } from '@shared/infra/identity-management/keycloak/service/keycloak-administration.service';
+import { IAccount, IAccountUpdate } from '@shared/domain';
+import { ObjectId } from '@mikro-orm/mongodb';
+import UserRepresentation from '@keycloak/keycloak-admin-client/lib/defs/userRepresentation';
 import { KeycloakIdentityManagementService } from './keycloak-identity-management.service';
+import { IKeycloakSettings, KeycloakSettings } from '../interface/keycloak-settings.interface';
+import { IdentityManagementService } from '../../identity-management.service';
 
-/**
- * This test does not run on CI!
- *
- * This test can be used to verify that schulcloud-server and Keycloak (ErWIn-IDM)
- * communication is working OK in the local developers environment.
- *
- * Once Keycloak is integrated into the CI setup, this test can be re-factored and enabled.
- *
- */
-xdescribe('KeycloakIdentityManagement', () => {
+describe('KeycloakIdentityManagementService Integration', () => {
```

Comment:
As in [account-idm.service.integration.spec.ts](https://github.com/hpi-schul-cloud/schulcloud-server/pull/3732/files#diff-04d1151b1f70b88a3601f2fc7267ef047a491aa4a2e669312f838bbe81aa3535) the tests (maybe the entire describe?) should be skipped if keycloak is not reachable.

Reply:
done

Reply:
No changes regarding this. Forgotten Push?